src/atoms/table/Table.selectors.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name": ".table--odd-even",
```

Comment:
We don't have any styles for it, are you missing commit something or we're waiting for updated component?

Reply:
Related styles are under `TableRow.scss`